### PR TITLE
Feature/creation ror application

### PR DESCRIPTION
**Pull request by CarlosSe**
Quick Info
---
Create basic Ruby on Rails application
Ruby: v3.1.0
Rails: v7.0.3.1

Migrations? :-1:

What does this change?
----
- Creates a Ruby on Rails application
- Configures CircleCI 

How do you manually test this?
----
Run on the command line console
> bundle exec rails server

then browse to http://127.0.0.1:3000 you'll see the RoR landing page

Screenshots
----

### Desktop
Screenshot 2022-08-22 at 11-05-44 Ruby on Rails 7 0 3 1

### CODE REVIEW

**Review by ccarruitero — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      RAILS_ENV: test
+    steps:
+      - checkout
+      - ruby/install-deps
+      - run:
+          name: Wait for DB
+          command: dockerize -wait tcp://localhost:5432 -timeout 1m
+      - run:
+          name: Database setup
+          command: bundle exec rails db:schema:load --trace
```
Comment (softr8) [inline .circleci/config.yml:39]: you're missing the step that actually run all the specs, something like: `bundle exec rspec spec/`

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  # gem "rack-mini-profiler"
+
+  # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
+  # gem "spring"
+end
+
+group :test do
+  # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
+  gem "capybara"
+  gem "selenium-webdriver"
```
Comment (softr8) [inline Gemfile]: replace selenium and webdrivers for https://github.com/rubycdp/cuprite, it's faster and more reliable, same API

File: test/test_helper.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+ENV["RAILS_ENV"] ||= "test"
+require_relative "../config/environment"
+require "rails/test_help"
+
+class ActiveSupport::TestCase
```
Comment (softr8) [inline test/test_helper.rb:5]: if you are planning to use just mini test, then add the capybara settings here, and at least one system test that opens homepage

File: .circleci/config.yml (hunk 1 above)
Comment (CarlosSe) [reply .circleci/config.yml:39]: In this stage RSpec is not installed, I'll create the step in the CI after integrating and configuring Solidus  😄

File: Gemfile (hunk 2 above)
Comment (CarlosSe) [reply Gemfile]: Thank you, I'll change selenium for Cuprite

File: test/test_helper.rb (hunk 3 above)
Comment (CarlosSe) [reply test/test_helper.rb:5]: Thank you I'm going to use RSpec, in the next PRs after integrating Solidus I'll configure FactoryBot and a Driver

**Pull request merged**